pkg/tasks/reset.go

```diff
@@ -29,6 +31,91 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 )
 
+// CleanupLoadBalancers Deletes all the load balancers services from the cluster.
+func CleanupLoadBalancers(s *state.State) error {
+	if !s.CleanupLoadBalancers {
+		return nil
+	}
+	var lastErr error
+	s.Logger.Infoln("Deleting load balancer services...")
+	_ = wait.ExponentialBackoff(defaultRetryBackoff(3), func() (bool, error) {
```

Comment:
We don't need exponential backoff here I think, each task already has 3 times backoff.